services/initChatAnonymous/initChatAnonymousServices.ts

```diff
@@ -0,0 +1,70 @@
+import { AxiosResponse } from "axios";
+import apiAnonymous from "@shared/fetcherAnonymous";
+
+type User = {
+  user_id: string;
+  human_id: string;
+  country_code: string;
+  username: string;
+  real_name: null;
+  last_active_at: string;
+  profile_pic_path: string;
+  profile_pic_asset_id: string;
+  profile_pic_public_id: string;
+  status: string;
+  bio: string;
+  is_banned: boolean;
+  is_anonymous: boolean;
+  allow_anon_dm: boolean;
+  only_received_dm_from_user_following: boolean;
+  is_backdoor_user: boolean;
+  verified_status: string;
+  anon_user_info_color_code: string;
+  anon_user_info_color_name: string;
+  anon_user_info_emoji_code: string;
+  anon_user_info_emoji_name: string;
+};
+
+type Message = {
+  id: string;
+  text: string;
+  html: string;
+  type: string;
+  user: any;
+  attachments: [];
+  latest_reactions: [];
+  own_reactions: [];
+  reaction_counts: {};
+  reaction_scores: {};
+  reply_count: number;
+  deleted_reply_count: number;
+  message: string;
+};
+
+type InitChatAnonymousResponse = {
+  message: Message;
+  durationg: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the `InitChatAnonymousResponse` type definition. It should be `duration` instead of `durationg`.
```diff
-  durationg: string;
+  duration: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->